### ISSUE TITLE
Errors related to ex_doc since version 3.3.1

### ISSUE THREAD

**Issue by davidjulien** [pma/amqp#234]
Hello,

When updating `amqp` from version 3.3.0 to version 3.3.1 in my project, dialyzer emits multiple warnings:
```
Error loading makeup_erlang, dependency list may be incomplete.
 {~c"no such file or directory", ~c"makeup_erlang.app"}
Error loading makeup_elixir, dependency list may be incomplete.
 {~c"no such file or directory", ~c"makeup_elixir.app"}
Error loading makeup, dependency list may be incomplete.
 {~c"no such file or directory", ~c"makeup.app"}
Error loading ex_doc, dependency list may be incomplete.
 {~c"no such file or directory", ~c"ex_doc.app"}
```

And tests don't run anymore:
```
** (Mix) Could not start application makeup: could not find application file: makeup.app
```

If I switch to `{:amqp, git: "https://github.com/pma/amqp.git"}` in my `mix.exs` file, it works correctly.

Maybe the hex package is not valid? The `mix.exs` file in `deps/amqp` with version 3.3.1 is not exactly the same than the `mix.exs` in your repo:
```
diff --color -r ./mix.exs ../../../../../../amqp/mix.exs
23c23
<       applications: [:amqp_client, :logger, :makeup, :makeup_elixir, :makeup_erlang, :ex_doc],
---
>       applications: [:amqp_client, :logger],
34d33
<       {:makeup, "1.1.0", only: :docs},
```

Maybe I missed something?

Regards

**Comment by ono** [comment]
Ah, what a pain... 🤦‍♂️ 

I just retired 3.3.1. I am really sorry about the hassle 🙇‍♂️ 

I also reopened #232 and will release 3.3.2 once all doc generation issue is sorted. 

Thank you very much for reporting. Really appreciate it ❤️

**Comment by ono** [comment]
Just released 3.3.2. Thanks again!

**Comment by davidjulien** [comment]
Thank you for your quick fix! It's working again with 3.3.2
